### ISSUE TITLE
docker-compose build fails because of depends_on: condition

### ISSUE THREAD

**Issue by Albirew** [syuilo/Misskey#9672]
## 💡 Summary

depends_on: condition appears to have been removed in version 3 of the docker compose specification, but then re-introduced in version 3.9.

## 🥰 Expected Behavior

docker-compose build finishing normally

## 🤬 Actual Behavior

docker-compose build fails at start
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.web.depends_on contains an invalid type, it should be an array
```

## 📝 Steps to Reproduce

1. follow https://misskey-hub.net/en/docs/install/docker.html instructions on either a new configuration or trying to upgrade an old one

## 📌 Environment

Misskey version: 13.0.0
Your OS: debian 11 with docker 20.10.22
Your browser: firefox 109

**Comment by ikuradon** [comment]
What version of docker-compose are you using? (`docker-compose --version`)
docker compose schema version 3.9 has been available since docker-compose>=1.27.1.

**Comment by Albirew** [comment]
I'm indeed an idiot who haven't updated my docker-compose since 1.24.1 (and is now discovering docker compose instead of docker-compose)...
